scripts/sct_register_to_template.py

```diff
@@ -261,6 +261,10 @@ def main(args=None):
     # check input labels
     labels = check_labels(fname_landmarks, label_type=label_type)
 
+    vertebral_alignement = False
```

Comment:
in English: vertebral_alignment, but that's a minor issue

Reply:
Changed